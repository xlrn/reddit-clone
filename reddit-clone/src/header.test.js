import renderer from 'react-test-renderer';
import Header from './header';
import { MemoryRouter } from 'react-router-dom';

it ('renders properly', () => {
    const component = renderer.create(
        <MemoryRouter>
            <Header></Header>
        </MemoryRouter>
    )
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

})