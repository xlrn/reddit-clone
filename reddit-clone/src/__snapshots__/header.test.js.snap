// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<nav
  className="navbar navbar-expand-lg navbar-light bg-light"
>
  <div
    className="col-4 d-flex"
  >
    <a
      className="navbar-brand mx-2"
      href="/"
    >
      Reddit
    </a>
    <form
      className="d-flex"
    >
      <input
        aria-label="Search"
        className="form-control"
        placeholder="Search"
        type="search"
      />
      <button
        className="btn btn-outline-success"
        type="submit"
      >
        Search
      </button>
    </form>
  </div>
  <div
    className="col-4 d-flex justify-content-between"
  >
    <a
      href="/hot"
      onClick={[Function]}
    >
      <button
        className="btn btn-primary"
        type="button"
      >
        Hot
      </button>
    </a>
    <a
      href="/new"
      onClick={[Function]}
    >
      <button
        className="btn btn-primary"
        type="button"
      >
        New
      </button>
    </a>
    <a
      href="/top"
      onClick={[Function]}
    >
      <button
        className="btn btn-primary"
        type="button"
      >
        Top
      </button>
    </a>
  </div>
</nav>
`;
